light_curves/lc_classifier.ipynb

```diff
@@ -0,0 +1,983 @@
+    "5. Pandas is user friendly\n",
+    "\n",
+    "\n",
+    "## Input\n",
+    "Light curve parquet file of multiband light curves from the mulitband_lc.ipynb demo notebook.  The format of the light curves is a Pandas multiindex data frame\n",
+    "\n",
+    "A useful reference for what sktime expects as input to its ML algorithms: https://github.com/sktime/sktime/blob/main/examples/AA_datatypes_and_datasets.ipynb\n",
+    "\n",
+    "## Output\n",
+    "Trained classifiers as well as estimates of their accuracy and plots of confusion matrices\n",
+    "\n",
+    "## Imports\n",
+    "- `pandas` to work with light curve data structure\n",
+    "- `numpy` for numerical calculations\n",
+    "- `matplotlib` for plotting\n",
+    "- `sys` for paths\n",
+    "- `astropy` to work with coordinates/units and data structures\n",
+    "- `tqdm` for showing progress meter\n",
+    "- `sktime` ML algorithms specifically for time-domain data\n",
+    "- `sklearn` general use ML algorthims with easy to use interface\n",
+    "\n",
+    "## Authors\n",
+    "\n",
+    "## Acknowledgements"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "!pip install sktime\n",
+    "!pip install tqdm\n",
+    "!pip install googledrivedownloader"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import sys\n",
+    "from astropy.time import Time\n",
+    "from tqdm import tqdm\n",
+    "from google_drive_downloader import GoogleDriveDownloader as gdd\n",
+    "\n",
+    "from sktime.registry import all_estimators\n",
+    "from sktime.registry import all_tags\n",
+    "\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.metrics import accuracy_score, confusion_matrix, ConfusionMatrixDisplay\n",
+    "\n",
+    "# local code imports\n",
+    "sys.path.append('code/')\n",
+    "from fluxconversions import mjd_to_jd\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Read in a dataset of archival light curves"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#access structure of light curves made in the light curve notebook\n",
+    "# has CLAGN & SDSS small sample, all bands\n",
+    "# https://drive.google.com/file/d/1y92XoKuUa-auAWM39KpSaBC9VpEqXcC6/view?usp=share_link\n",
+    "\n",
+    "gdd.download_file_from_google_drive(file_id='1y92XoKuUa-auAWM39KpSaBC9VpEqXcC6',\n",
+    "                                    dest_path='./data/df_lc_458sample.parquet.gzip',\n",
+    "                                    unzip=True)\n",
+    "\n",
```

Comment:
If we run multibandlc notebook, the idea is to have the dataframe saved in output instead of data. Shouldn't we then also download this parquet into output?

Reply:
I think this parquet should be in data/ because it is required to run the lc_classifier notebook.  But this reminds me I also wanted to get rid of that .gzip

Reply:
The output of one notebook is the input to the other, so a bit confusing to me.